### ISSUE TITLE
Right click context menu does not appear

### ISSUE THREAD

**Issue by MartijnMink** [munyabe/FormatAllFiles#4]
Thanks for making this.

I am using Microsoft Visual Studio Community 2015, Version 14.0.25431.01 Update 3.

I installed successfully (see install_log.txt) and restarted visual studio. The extension is shown in Tools > Extensions and Updates, but the right click context menu is not appearing when i right click on a source folder, project or solution. Do you know what i did wrong or how i can make it work?

Thanks,
Martijn

[install_log.txt](https://github.com/munyabe/FormatAllFiles/files/871724/install_log.txt)

![schermafdruk 2017-03-27 09 34 23](https://cloud.githubusercontent.com/assets/26625646/24345460/9524797e-12d0-11e7-8138-5d22f8f1e0c7.png)

**Comment by MartijnMink** [comment]
The issue has disappeared, the extension now works perfectly. I did two things which could have caused it 

1) I installed the extension Format document on Save [https://marketplace.visualstudio.com/items?itemName=mynkow.FormatdocumentonSave]

2) I killed explorer.exe from the task manager and then restarted. Maybe this has a similar effect to doing a 'complete' shutdown/restart instead of the default 'hibernate shutdown' (i use windows 10).

I would expect "1" would have no effect, but i mention it anyway for completeness.
